### PR TITLE
possibly unused files

### PR DESCRIPTION
**Pull request by n-hutton**

### CODE REVIEW

File: libs/core/include/core/containers/mapping.hpp [hunk 1]
```diff
@@ -1,111 +0,0 @@
-#pragma once
```
Comment (ejfitzgerald) [inline libs/core/include/core/containers/mapping.hpp]: Can we keep this please. This might not be useful currently, however, very useful in general

**Review by ejfitzgerald — CHANGES_REQUESTED** [review]
I would like to keep the mapping an suggest @baykaner will want to keep some of the maths / ml stuff at least

File: libs/core/include/core/containers/mapping.hpp (hunk 1 above)
Comment (n-hutton) [reply libs/core/include/core/containers/mapping.hpp]: yup

**Review by baykaner — DISMISSED** [review]
don't remove the math files. I will raise a task to write tests for them.
Also, I need konstantinos to weigh in on the vectorise fiels.

👍

File: libs/network/include/network/muddle/rpc/server.hpp [hunk 2]
```diff
@@ -1,154 +0,0 @@
-#pragma once
```
Comment (ejfitzgerald) [inline libs/network/include/network/muddle/rpc/server.hpp:1]: When did these get reintroduced have we had a merge blunder somewhere?

File: libs/miner/include/miner/block_optimiser.hpp [hunk 3]
```diff
@@ -1,457 +0,0 @@
-#pragma once
```
Comment (ejfitzgerald) [inline libs/miner/include/miner/block_optimiser.hpp]: Can we keep the annealer miner stuff. We have an open task to reinstate this

**Review by ejfitzgerald — CHANGES_REQUESTED** [review]

File: libs/network/include/network/muddle/rpc/server.hpp (hunk 2 above)
Comment (n-hutton) [reply libs/network/include/network/muddle/rpc/server.hpp:1]: You may have introduced it in commit 1bc800ac

File: libs/miner/include/miner/block_optimiser.hpp (hunk 3 above)
Comment (n-hutton) [reply libs/miner/include/miner/block_optimiser.hpp]: The following files have been saved from certain doom: 
libs/miner/include/miner/annealer_miner.hpp
 libs/miner/include/miner/block_optimiser.hpp
 libs/miner/include/miner/optimisation/binary_annealer.hpp

Comment (ejfitzgerald) [reply libs/miner/include/miner/block_optimiser.hpp]: 👍

File: libs/network/include/network/muddle/rpc/server.hpp (hunk 2 above)
Comment (ejfitzgerald) [reply libs/network/include/network/muddle/rpc/server.hpp:1]: Oh the nostalgia - that was when it originally came into `master` I was pretty sure that I binned / moved these files around with Muddle V2. Oh well you got the stragglers! 😁

File: libs/miner/include/miner/instance/binary_problem.hpp [hunk 4]
```diff
@@ -1,201 +0,0 @@
-#pragma once
```
Comment (ejfitzgerald) [inline libs/miner/include/miner/instance/binary_problem.hpp]: Please can we save this one too

**Review by ejfitzgerald — CHANGES_REQUESTED** [review]
Just one more file to save and then lets let @fetch-bot do its thang!

File: libs/miner/include/miner/instance/binary_problem.hpp (hunk 4 above)
Comment (n-hutton) [reply libs/miner/include/miner/instance/binary_problem.hpp]: Yup done

**Review by ejfitzgerald — APPROVED** [review]

**Pull request merged**